frontend/src/containers/Faq.js

```diff
@@ -75,6 +75,24 @@ export class Faq extends Component {
           <h3>How do I make a donation?</h3>
           <p>You rock! We are delighted you are asking this question. Go to the Collective’s Public Page (e.g. https://opencollective.com/wwcodeatl), choose your recurring donation amount and hit the Back Us or Donate button, fill out the form et voilà! Now you are part of that collective, your community will appreciate it.</p>
           
+          <h3>How do I submit an expense?</h3>
+          <p>Go to your collective page to submit an expense. You will need to upload a valid image for it to get reimbursed. Don’t forget to include your paypal email if it’s different from your user’s email.</p>
+          <p>There are two ways of using your Collective’s money:
+            <ul>
+              <li>Submit a receipt for a service you paid for (online services, meals, plane tickets).</li>
```

Comment:
Submit a receipt for something you paid for like online services, meals, plane tickets, etc.